ada_feeding/ada_feeding/behaviors/ros/time.py

```diff
@@ -1,5 +1,7 @@
-#!/usr/bin/env python3
```

Comment:
For this and others, are there any instances where our existing stack calls Python files with `./`? My understanding is this would be the only reason to have the shebang line present, so I'm wondering if it was done intentionally. Anecdotally, I haven't seen any such cases, but it's good to double check.

Reply:
[ROS uses it for running Python nodes](https://docs.ros.org/en/humble/How-To-Guides/Installation-Troubleshooting.html#failed-to-create-process), I think to allow node creators to specify python2 vs python3.